js/index.js

```diff
@@ -1,491 +0,0 @@
-let beltSpeed = 2.5; 
-let beltX = 0;
-let randomARR = [];
-let x
-let y
-let w
-let h
-let randomArrW = 0 
-let loadedPackage = []
-let typedText =""
-let insideW
-let insideX
-let leftW
-let leftH
-let messageVisible = false;
-let gameState = 0;
-let button;
-let restartButton;
-let score = 0
-let messageLevelUp = false;
-let level = 2500
-let song;
-let errorSound;
-let gameOverSound;
-let loadedSound;
-// let stringArr = []
-// let randomText = ""
-
-
-setInterval(() => {if(gameState === 1){
-  randomPackage(packages);
-  organizePackage()}
-}, level); 
-
-
-function preload() {
-  // bgImg = loadImage('https://t3.ftcdn.net/jpg/00/88/98/18/360_F_88981880_YjJManMJ6hJmKr5CZteFJAkEzXIh8mxW.jpg');
-  bgImg = loadImage("img/background.png")
-  song = createAudio("sounds/music.mp3")
-  errorSound = createAudio("sounds/error.mp3")
-  gameOverSound = createAudio("sounds/game-over.mp3")
-  loadedSound = createAudio("sounds/loaded.mp3")
-}
-
-function setup() {
-    createCanvas(windowWidth-5, windowHeight-5);
-    button = createButton('Start Game');
-    button.position(width / 2 - button.width / 2 - 60 , height / 2 + 70);
-    button.mousePressed(startGame);
-    button.mouseOver(hover);
-    button.mouseOut(endHover);
-    button.style('font-size', '30px');
-    button.style('font-weight', 'bold')
-    button.style('color', color(213,0,41));
-    button.style('background-color', color(254,204,0));
-    button.style('border', 0);
-    button.style('padding', "10px");
-    button.style('box-shadow', "3px 4px #888888");
-
-
-    restartButton = createButton('Restart Game');
-    restartButton.position(width / 2 - restartButton.width / 2 , height / 2 + 40);
-    restartButton.mousePressed(restartGame);
-    restartButton.mouseOver(hover);
-    restartButton.mouseOut(endHover);
-    restartButton.style('font-size', '30px');
-    restartButton.style('font-weight', 'bold')
-    restartButton.style('color', color(213,0,41));
-    restartButton.style('background-color', color(254,204,0));
-    restartButton.style('border', 0);
-    restartButton.style('padding', "10px");
-    restartButton.style('box-shadow', "3px 4px #888888");
```

Comment:
To keep the code DRY, you could have created a function that returns a button with these variables since they are mostly identical, and pass the callback function as an arg